src/js/tech/tech.js

```diff
@@ -313,6 +345,30 @@ class Tech extends Component {
     }));
   }
 
+
+  /**
+   * Initialize audiotrack listeners
+   *
+   * @method initAudioTrackListeners
+   */
+  initAudioTrackListeners() {
```

Comment:
Probably, as part of the release with this PR, I think we should introduce the stability index which would affect this method as well. We probably should have it log a deprecation warning.